nubis/bin/packer-bootstrap

```diff
@@ -87,32 +87,24 @@ fi
 sudo mkdir -p /etc/puppet
 sudo touch /etc/puppet/hiera.yaml
 
-# We have a file provisioner in main.json that'll copy over a .tar.gz of the librarian-puppet tree that's
-# created by make. We need to extract that so we can have a full puppet checkout ready for additional build
-# iterations.
-if [[ -f /tmp/librarian-puppet.tar.gz ]]; then
-   sudo tar -C / -zxf /tmp/librarian-puppet.tar.gz 2>&1
-   rm -f /tmp/librarian-puppet.tar.gz
-
```

Comment:
This ugly diff simply removes these two lines regarding the tar.gz file as well as removing the surrounding conditional.